types/robust-segment-intersect/robust-segment-intersect-tests.ts

```diff
@@ -0,0 +1,10 @@
+import crosses = require('robust-segment-intersect');
+type Coordinate = [number, number];
+function robustsegmentintersect(
+    a0: Coordinate,
+    a1: Coordinate,
+    b0: Coordinate,
+    b1: Coordinate
+): boolean {
+    return crosses(a0, a1, b0, b1);
+}
```

Comment:
```suggestion
const a0 = [-1, 0]
const a1 = [ 1, 0]
const b0 = [ 0,-1]
const b1 = [ 0, 1]

// $ExpectType boolean
crosses(a0, a1, b0, b1);
```
a test like actual usage?